std_msgs/Header header
int16 drone_id
geometry_msgs/Point odom